frontend/src/components/last-passed.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { Badge } from '@patternfly/react-core';
+import { HttpClient } from '../services/http';
+import { Settings } from '../settings';
+
+import {
+  buildParams,
+  toAPIFilter,
+} from '../utilities';
+
+const LastPassed = (props) => {
+  const {filters} = props;
+
+  const [resultData, setResultData] = useState();
+
+  useEffect(() => {
+    // get the passed/failed/etc test summary
+    // disregard result filter so we can filter on last passed
+    if (filters) {
+      let params = {...filters};
```

Comment:
Nitpick: Could be const since we are not re-assigning object but changing it's props